atlasdb-jepsen-tests/src/main/java/com/palantir/atlasdb/jepsen/PartitionByInvokeNameCheckerHelper.java

```diff
@@ -0,0 +1,78 @@
+/**
+ * Copyright 2017 Palantir Technologies
+ *
+ * Licensed under the BSD-3 License (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://opensource.org/licenses/BSD-3-Clause
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.palantir.atlasdb.jepsen;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.function.Supplier;
+import java.util.stream.Collectors;
+
+import com.palantir.atlasdb.jepsen.events.Checker;
+import com.palantir.atlasdb.jepsen.events.Event;
+import com.palantir.atlasdb.jepsen.events.InvokeEvent;
+
+public class PartitionByInvokeNameCheckerHelper implements Checker {
+
+    private Supplier<Checker> checkerSupplier;
+
+    PartitionByInvokeNameCheckerHelper(Supplier<Checker> checkerSupplier) {
```

Comment:
Hmm, the package visibility looks a little weird. I'd like to check that this is intentional.